source/lib/tabs-service.js

```diff
@@ -26,8 +26,8 @@ export async function openTab(url) {
 	const permissionGranted = await queryPermission('tabs');
 	if (reuseTabs && permissionGranted) {
 		const matchingUrls = [url];
-		if (url.endsWith('/notifications')) {
-			matchingUrls.push(url + '?query=is%3Aunread');
+		if (new URL(url).pathname.endsWith('/notifications')) {
+			matchingUrls.push(url + '?query=*');
```

Comment:
The canonical link to the notifications page is `https://github.com/notifications`, not `https://github.com/notifications?query=` which you could be using here.

The code looks for any tabs that have `/notifications` or `/notifications?query=is%3Aunread`.

The actual fix here should be adding another matching URL to look for.

```suggestion
			matchingUrls.push(url + '?query=');
			matchingUrls.push(url + '?query=is%3Aunread');
```

This also does not require you to use the `new URL(url).pathname`, the param being passed to this function would always be a constant.

Reply:
applied in [`1e30721` (#299)](https://github.com/sindresorhus/notifier-for-github/pull/299/commits/1e307217a11db749fd8205183005fd001e88b126).